libs/dal/src/repositories/integration/integration.schema.ts

```diff
@@ -0,0 +1,35 @@
+import * as mongoose from 'mongoose';
+import { Schema, Document } from 'mongoose';
+import { schemaOptions } from '../schema-default.options';
+import { IntegrationEntity } from './integration.entity';
+
+const integrationSchema = new Schema(
+  {
+    _applicationId: {
+      type: Schema.Types.ObjectId,
+      ref: 'Application',
+      index: true,
+    },
+    _organizationId: {
+      type: Schema.Types.ObjectId,
+      ref: 'Organization',
+      index: true,
+    },
+    providerId: Schema.Types.String,
+    channel: Schema.Types.String,
```

Comment:
I'm missing in the data model the "active" key right?

Reply:
You are totally right! :)